docs/denoising.rst

```diff
@@ -0,0 +1,352 @@
+##############################
+Denoising Data with Components
+##############################
+
+Decomposition-based denoising methods like ``tedana`` will produce two important outputs: component time series and component classifications.
+The component classifications will indicate whether each componet is "good" (accepted) or "bad" (rejected).
+To remove noise from your data, you can regress the "bad" components out of it, though there are multiple ways to accomplish this.
+
+By default, ``tedana`` will perform a regression including both "good" and "bad" components, and then will selectively remove the "bad" components from the data.
+This is colloquially known as "non-aggressive denoising".
+
+However, users may wish to apply a different type of denoising, or to incorporate other regressors into their denoising step, and we will discuss these alternatives here.
+
+.. admonition:: Independence vs. Orthogonality
+
+  ``tedana`` uses independent component analysis (ICA) to decompose the data into time series that are assumed to reflect meaningful underlying signals.
+  It then classifies each time series (a.k.a. component) as "accepted" (BOLD-like) or "rejected" (non-BOLD-like).
+
+  The components from an ICA are `statistically independent`, rather than orthogonal.
+  Orthogonal time series are uncorrected, so there is no shared variance.
+  With independent components, however, the time series may be correlated with one another, and thus share variance.
+  If a rejected component shares meaningful variance with an accepted component,
+  then regressing the rejected components from your data may also remove meaningful signal associated with the accepted component as well.
+  This conundrum is what motivates the different approaches used for decomposition-based denoising methods described here.
```

Comment:
```suggestion
  ``tedana`` uses independent component analysis (ICA) to decompose the data into components which each have a spatial map of weights and time series that are assumed to reflect meaningful underlying signals.
  It then classifies each component) as "accepted" (BOLD-like) or "rejected" (non-BOLD-like). The data are denoised by taking the time series of the rejected components and regressing them from the data.

  ``tedana`` uses a spatial ICA, which means the components are `statistically independent` across space, not time. That means the time series from accepted and rejected components can share variance.
  If a rejected component shares meaningful variance with an accepted component,
  then regressing the rejected components' time series from your data may also remove meaningful signal associated with the accepted component as well.
  Depending on the application, people may take different approaches on how to handle variance that is shared between accepted and rejected components. These different approaches towards decomposition-based denoising methods are described here.
```
Maybe I shouldn't be reading this while tired, but I don't think this quite explains the issue. It's not about statistical independence vs orthogonality. It's that we're doing a spatial ICA that results in components that are  statistically independent across space. The distinction between orthogonality and statistical independence is not relevant to this specific issue.

Reply:
I've added your text in 6b3cb00. Thanks!